addons/spreadsheet/static/tests/global_filters/global_filters_model_test.js

```diff
@@ -2386,5 +2405,7 @@ QUnit.module("spreadsheet > Global filters model", {}, () => {
 
         moveGlobalFilter(model, lastYearFilterId, -2);
         assert.deepEqual(model.getters.getGlobalFilters()[0].id, lastYearFilterId);
+
+        model.dispatch("UNDO", {});
```

Comment:
Should be `REQUEST_UNDO`. Or even better `undo(model)`.
It's the reason why the test is red

Reply:
oups, I left that to test, but the other test covers more cases :)